modules/jooby-bom/pom.xml

```diff
@@ -12,7 +12,7 @@
   <properties>
   <jooby.version>1.2.3</jooby.version>
   <funzy.version>0.1.0</funzy.version>
-  <jackson.version>2.9.2</jackson.version>
+  <jackson.version>2.9.4</jackson.version>
```

Comment:
bom file is auto-generated at release time, no need to update it manually :)

Reply:
Grin - then we shouldn't be checking it in :-P